docker/Dockerfile

```diff
@@ -49,6 +49,15 @@ COPY requirements/project-requirements.txt $PYTHONPATH/project-requirements.txt
 COPY requirements/certego-requirements.txt $PYTHONPATH/certego-requirements.txt
 WORKDIR $PYTHONPATH
 
+
+# Installing dependencies for zippy_scan file analyzer
+RUN git clone https://github.com/thinkst/zippy \
+&& cd zippy && python3 setup.py build \
+&& python3 setup.py sdist \
+&& pip3 install dist/*.tar.gz
+
+RUN rm -r zippy
```

Comment:
merge this in a single RUN to reduce the layers of the image + move those before the COPY commands otherwise this will be executed _again_ every time we change something in the python requirements

Reply:
on it!